From 223851b68fedad730747652bc6560e40ca695ea7 Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Mon, 07 Mar 2011 13:54:53 +0000
Subject: Use XCB_ATOM_* atom names rather than deprecated atom names

Those deprecated atom names were removed from xcb/util in commit
2f0334b3eb49fa3a0d6daf6b9dde10e480a5c59f

Found by tinderbox, see:

http://tinderbox.freedesktop.org/builds/2011-03-06-0008/logs/xlsclients/#build
http://tinderbox.freedesktop.org/builds/2011-03-07-0001/logs/xlsclients/#build

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
Tested-by: Dan Nicholson <dbn.lists@gmail.com>
Signed-off-by: Peter Harris <pharris@opentext.com>
---
diff --git a/xlsclients.c b/xlsclients.c
index 3a34865..15d3f1a 100644
--- a/xlsclients.c
+++ b/xlsclients.c
@@ -539,21 +539,21 @@ print_client_properties(xcb_connection_t *dpy, xcb_window_t w, int verbose, int
      * get the WM_CLIENT_MACHINE and WM_COMMAND list of strings
      */
     cs->client_machine = xcb_get_property(dpy, 0, w,
-			    WM_CLIENT_MACHINE, XCB_GET_PROPERTY_TYPE_ANY,
+			    XCB_ATOM_WM_CLIENT_MACHINE, XCB_GET_PROPERTY_TYPE_ANY,
 			    0, 1000000L);
     cs->command = xcb_get_property(dpy, 0, w,
-			    WM_COMMAND, XCB_GET_PROPERTY_TYPE_ANY,
+			    XCB_ATOM_WM_COMMAND, XCB_GET_PROPERTY_TYPE_ANY,
 			    0, 1000000L);
 
     if (verbose) {
 	cs->name = xcb_get_property(dpy, 0, w,
-			    WM_NAME, XCB_GET_PROPERTY_TYPE_ANY,
+			    XCB_ATOM_WM_NAME, XCB_GET_PROPERTY_TYPE_ANY,
 			    0, 1000000L);
 	cs->icon_name = xcb_get_property(dpy, 0, w,
-			    WM_ICON_NAME, XCB_GET_PROPERTY_TYPE_ANY,
+			    XCB_ATOM_WM_ICON_NAME, XCB_GET_PROPERTY_TYPE_ANY,
 			    0, 1000000L);
 	cs->wm_class = xcb_get_property(dpy, 0, w,
-			    WM_CLASS, STRING,
+			    XCB_ATOM_WM_CLASS, XCB_ATOM_STRING,
 			    0, 1000000L);
     }
 
@@ -569,7 +569,7 @@ print_text_field(xcb_connection_t *dpy, char *s, xcb_get_property_reply_t *tp)
     }
 
     if (s) printf ("%s", s);
-    if (tp->type == STRING && tp->format == 8) {
+    if (tp->type == XCB_ATOM_STRING && tp->format == 8) {
 	printf ("%.*s", (int)tp->value_len, (char *)xcb_get_property_value(tp));
     } else {
 	unknown (dpy, tp->type, tp->format);
--
cgit v0.8.3-6-g21f6
